### PR TITLE
Create testcases

### LINKED ISSUE #207: Create Testcases
for unit test, system test and espresso test

### PR DESCRIPTION
**Pull request by MBRzealand**
Closes #207

### CODE REVIEW

**Review by Hjordrup — APPROVED** [review]
This all looks good

File: 05 Test/EspressoTest.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    <td>          Width  = 10.7                                     </td>
+    <td>          Displays: 
+                  "Bredden er for lille, min: 10.8"                 </td>
+    <td>          Displays: 
+                  "Bredden er for lille, min: 10.8"                 </td>
+    <td>          　✓                                               </td>
+  </tr>
+  <tr>
+    <td>          Test_06 </td>
+    <td>          Height too large exception                        </td>
```
Comment (Bombohog) [inline 05 Test/EspressoTest.md:72]: "too high" and not "too large"

File: 05 Test/UnitTest.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    <td>          Width too small exception                         </td>
+    <td>          1. Input wall width                               </td>
+    <td>          Width  = 10.7                                     </td>
+    <td>          Bredden er for lille, min: 10.8                   </td>
+    <td>          Bredden er for lille, min: 10.8                   </td>
+    <td>          　✓                                               </td>
+  </tr>
+  <tr>
+    <td>          Test_06                                           </td>
+    <td>          Height too large exception                        </td>
```
Comment (Bombohog) [inline 05 Test/UnitTest.md:64]: "too high" and not "too large"

**Review by Bombohog — COMMENTED** [review]
These should have been changed, but eh.

**Pull request merged**